app-generated-skeleton/threshold.properties

```diff
@@ -1,5 +1,5 @@
 # With all extensions enabled, it could take the dev mode a long time to start.
```

Comment:
This is not a good trend, hope it's GHActions env related

Created https://trello.com/c/e3MxfB6b/913-check-performance-111-vs-21 to double check the results

Reply:
I'm sure this is a GH action env related... as it didn't happen all the times